packages/vira/src/Vira/State/Type.hs

```diff
@@ -157,7 +147,6 @@ $(deriveSafeCopy 0 'base ''RepoName)
 $(deriveSafeCopy 0 'base ''JobId)
 $(deriveSafeCopy 0 'base ''Job)
 $(deriveSafeCopy 0 'base ''Branch)
-$(deriveSafeCopy 0 'base ''RepoSettings)
```

Comment:
Once this PR is merged, existing state directories will be rejected:

<img width="572" height="178" alt="Image" src="https://github.com/user-attachments/assets/0105a18b-72b9-4a3d-a8e5-3ee24e57434a" />

---

What is the strategy for upgrading existing instances? 

(I don't think we should do acid-state migration before v1.)